### ISSUE TITLE
YAKUAKE:  panel icon DROPDOWN DOESNT WORK

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-next#97]

**Comment by UriHerrera** [comment]
For an inexplicable reason, Plasma keeps creating duplicate launchers for both embedded desktop launchers in the panel. This occurs every time Plasma is executed.

![Captura de pantalla de 2024-05-15 13-55-05](https://github.com/blue-systems/netrunner-next/assets/3053525/35a0bc21-e1cd-4194-8317-2fd39cc7fffb)

![Captura de pantalla de 2024-05-15 13-59-18](https://github.com/blue-systems/netrunner-next/assets/3053525/15dd5828-9dce-4b55-b6ce-a8c65c6d1aa7)

The panel configuration has not changed in this regard, and it seems to be a bug in Plasma or a KDE Framework.

![2024-05-15_14-28](https://github.com/blue-systems/netrunner-next/assets/3053525/bc37faeb-2c0f-41ac-8f17-1daa65da9463)

As is, this bug will continue to break the embedded launcher for Yakuake. On the other hand, the embedded launcher for Spectacle keeps working, regardless of this bug in Plasma. I'd assume that embedding the launchers in the panel is causing the problem in the first place.

That being the case, I recommend using a widget/plasmoid called _Configurable Button_ instead. This widget will allow us to include a launcher for Yakuake (and Spectacle or whatever application or action, like triggering a KWin effect) without having this problem. Best of all, it works the same way without triggering the bug. One-click opens Yakuake; another click closes it.

The only difference is that since the icon is pulled from the Breeze icon theme, the icon theme provides an icon for Yakuake that is for smaller sizes that look different.

![Captura de pantalla de 2024-05-15 14-08-30](https://github.com/blue-systems/netrunner-next/assets/3053525/ffb43e41-8eaf-4945-98c6-4e70adc11f15)

In summary, other than the icon, going this route will work around that bug and provide the same functionality.

![Captura de pantalla de 2024-05-15 14-20-22](https://github.com/blue-systems/netrunner-next/assets/3053525/b603fda1-9b96-4a01-8125-abd24415867d)
